### ISSUE TITLE
Research tab cleanup

### ISSUE THREAD

**Issue by Seanny123** [ctn-waterloo/website#48]
A lot of stuff under the "Research Tab" is either [broken](https://github.com/ctn-waterloo/website/issues/31), so out of date that it's useless or really awkwardly placed.

**Files that are worth saving:**
 - "Software" section
 - nef/statistical-inference.md (this isn't linked to anymore from the base NEF page)
- "Theoretical Neuroscience" section

**Out-of-date files to be archived:**
- "Perception" section (this project hasn't been worked on since 2009 and Charlie Tang has been gone for a while now)
- "Cognition" section (seems to be filled with obsolete projects)
 - cognition/neural-cognitive-architecture.md (are we even still trying to re-implement ACT-R?)
 - cognition/modelling-problem-solving-in-ravens-progressive-matrices.md (this project has been completed)
 - cognition/cognitive-robotics.md (this project has been abandoned by the person noted and has really morphed into a totally different thing)
- "Motor Control" section (has been complete by Travis)
- "Other" section

**Should be moved, but I'm not sure where and I'm afraid of breaking other links:**
- Neural Modelling Constants and Constraints

May I delete from the website the files that aren't worth saving and fix the ones that are?

If any of these project sections (which seem to be an earlier attempt at the lab blow) will be missed, may I recommend the alternative of somehow tagging our publications to automatically generate these sections so they don't go out of date?

As for the method of archiving, is the fact that these are contained on GitHub good enough or do we want some sort of hidden link so that they're still a part of the website? Do we need redirect links so that ancient links to these sections aren't broken?

**Comment by tbekolay** [comment]
Probably something to talk about at a lab meeting? As far as Nengo stuff is concerned, the Software stuff should definitely be moved to the appropriate Nengo repo (you can make a PR on Nengo 1.4 with the markdown files unchanged in it, just so we have them somewhere but you don't waste a lot of time trying to incorporate them in the current docs).

Not sure where the modelling constants and constraints should go, but they're linked to in a few places, so at the very least they should redirect to their new location. Probably easiest to keep them where they are though...

**Comment by Seanny123** [comment]
Indeed, I've already entered it into the schedule of the lab meeting!

FYI, for redirecting links, [there's a repository for that](https://github.com/jekyll/jekyll-redirect-from).

**Comment by tbekolay** [comment]
> FYI, for redirecting links, there's a repository for that.

This repo doesn't use Jekyll; however, I built in the ability to [easily redirect pages](https://github.com/ctn-waterloo/website/blob/master/ctn_waterloo/__init__.py#L182) since there were links to an old version of compneuro.uwaterloo.ca published in various places.

**Comment by Seanny123** [comment]
Gah, I always get Jekyll confused with Flask. My bad.

**Comment by Seanny123** [comment]
Here's my plan so far, which we can discuss during the meeting:
- Put the old tutorials into `nengo_1.4/simulator-ui/docs/`
- Put the old "Advanced Tips and Tricks" into `nengo_1.4/simulator-ui/docs/advanced/`
- Delete the NESim documentation ref, since the link to `http://compneuro.uwaterloo.ca/codelibrary/NEHelp/index.html` is dead and gone.
- Change the text in `software/software/nengo-model-archive.html` to note that this archive is only for Nengo 1.4 and that we are in the process of creating a new searchable archive
- Delete the old research-blog-esque files (they'll live on in the GitHub history) from the research tab
- Leave "Neural Modelling Constants and Constraints" and "Theoretical Neuroscience" untouched for now
- Update `statistical-inference.md` and re-include it in the NEF section.
 - Reference the actual section in the NEF book
 - What's the normalization publication referenced on that page?
 - Do we have more recent statistical inference publications that we could reference?

**Comment by Seanny123** [comment]
During the CTN meeting the following things were concluded:
- Yes, that page is super out of date.
- The content should be replaced, not removed and rewritten with the intent of being the "elevator pitch" of our lab.
 - It is unclear who will rewrite and maintain this section to keep it from going out of date again. I don't want to randomly distribute it to people like other chores, because I know some people are way faster writers than others. Just like some people are way faster developers than others, so we don't randomly assign development tasks. Consequently, it feels extremely likely that I will be the only one maintaining that tab by pinging the people who's research domain I am summarising. This does not seem horrible.
- The organisation of the tab should be changed and divided into sections: "Theory", "Applications" and "Tools".
- The Nengo Wikipedia page (if only the links) should be updated as well.
- The NESim information should not be lost to the dusts of time. We should figure out where that repository and documentation saved and make it more easily accessible.
